superset-frontend/src/dashboard/components/gridComponents/Column.jsx

```diff
@@ -58,6 +59,50 @@ const propTypes = {
 
 const defaultProps = {};
 
+const ColumnStyles = styled.div`
+  &.grid-column {
+    width: 100%;
+    position: relative;
+
+    & > :not(.hover-menu):not(:last-child) {
+      margin-bottom: ${({ theme }) => theme.gridUnit * 4}px;
+    }
+  }
+
+  &.grid-column--empty {
+    min-height: ${({ theme }) => theme.gridUnit * 25}px;
+
+    &:before {
+      content: 'Empty column';
```

Comment:
We need to translate the text here.

Reply:
We need to move that text out of `::before` as simply doing `content: ${t('Empty column')};` doesn't work